### PR TITLE
models bulk location validation

### PR DESCRIPTION
**Pull request by benauriemma**

### CODE REVIEW

File: rackcity/views/it_model_views.py [hunk 1]
```diff
@@ -241,6 +269,15 @@ def model_bulk_upload(request):
         except ObjectDoesNotExist:
             models_to_add.append(model_serializer)
         else:
+            if not model_height_change_valid(model_data, existing_model):
+                failure_message = \
+                    "Height change of this model causes conflicts: " + \
+                    "vendor="+model_data['vendor'] + \
+                    ", model_number="+model_data['model_number']
```
Comment (juliasaveliff) [inline rackcity/views/it_model_views.py]: nice failure message

File: rackcity/views/it_model_views.py [hunk 2]
```diff
@@ -83,6 +93,24 @@ def model_modify(request):
...
+def model_height_change_valid(new_model_data, existing_model):
+    if 'height' not in new_model_data:
+        return True
+    if new_model_data['height'] <= existing_model.height:
+        return True
+    else:
+        instances = ITInstance.objects.filter(model=existing_model.id)
+        for instance in instances:
+            if is_location_full(
+                instance.rack,
```
Comment (juliasaveliff) [inline rackcity/views/it_model_views.py]: addressed this in the original PR, but make sure this is `instance.rack.id` when you merge

File: rackcity/views/it_model_views.py (hunk 1 above)
Comment (benauriemma) [reply rackcity/views/it_model_views.py]: ty

File: rackcity/views/it_model_views.py (hunk 2 above)
Comment (benauriemma) [reply rackcity/views/it_model_views.py]: Weird it seems to work either way but you are correct

**Pull request merged**